trieste/models/keras/builders.py

```diff
@@ -82,39 +81,51 @@ def build_vanilla_keras_ensemble(
 
     return keras_ensemble
 
+
 def build_vanilla_keras_mcdropout(
     data: Dataset,
-    num_hidden_layers: int = 3,
-    units: int = 50,
+    num_hidden_layers: int = 5,
+    units: int = 300,
```

Comment:
This was just what seemed to work well when doing the integration test to see how well it could learn the function with a certain dropout rate. More testing might need to be done to find parameters that might be smaller, but it needs to be balanced with dropout rate. 

A smaller network has to have a smaller dropout rate to be able to still learn the function. A smaller dropout rate seems to sometimes lead to underexploration which then makes it slower even though the network took longer to train. 

For example: 

with EI using a rate of 0.1 and the above network size, seems like it has started moving towards the optimum but has slowed down due to the valley and is exploring too slowly potentially. :
![dropout_EI_01_fit](https://user-images.githubusercontent.com/88747458/163408904-e72ac83b-cdb1-45a4-ad84-af1a8af07121.png)

Then using a rate of 0.2:
![dropout_EI_02_fit](https://user-images.githubusercontent.com/88747458/163408963-5241038c-e8b6-4fa7-9939-6ea70d28140e.png)

Then using a rate of 0.25 and unable to learn the function:
![dropout_EI_025_fit](https://user-images.githubusercontent.com/88747458/163409034-9334e78d-d4e0-4a3c-9342-ee7087148265.png)

But yes, I think more testing needs to be done to find better defaults potentially and optimize it to the best speed. I kind of thought that would be part of benchmarking and trying different parameters.

Reply:
with integration test you mean BO integration test or learning the function, i.e. prediction test?

Reply:
I meant prediction test. I did try a few different parameters in the BO test as well and that was what seemed to work well for now, but potentially need to do more testing to find better defaults.